pkg/plugin/plugin.go

```diff
@@ -174,7 +182,7 @@ func (p *plugin) RecoverEtcdCluster(ctx context.Context, cs *api.OpenShiftManage
 	if err := clusterUpgrader.EtcdRestoreDeleteMasterScaleSet(ctx); err != nil {
 		return err
 	}
-	err = deployFn(ctx, azuretemplate)
+	_, err = deployFn(ctx, azuretemplate)
```

Comment:
can the return value be safely discarded?  Is this a bug waiting to happen?

Reply:
TODO